apidoc/Titanium/Calendar/Event.yml

```diff
@@ -246,13 +250,19 @@ properties:
         since: "3.1.3"
         removed: "3.2.0"
     platforms: [iphone, ipad]
-    
+
   - name: recurrenceRules
     summary: The recurrence rules for the calendar item. (Available in iOS 5.1 and above.)
     type: Array<Titanium.Calendar.RecurrenceRule>
     osver: {ios: {min: "5.0"}}
     platforms: [iphone, ipad]
 
+  - name: rrule
```

Comment:
The `rrule` property is not defined in the Android `EventProxy`.

Reply:
Also, iOS' `recurrenceRules` property appears to be "RFC 2445" as well. I think we should dump this property and try to make it portable with the existing rules property for portability.